### PR TITLE
get 100% coverage for itemscontroller

### PR DESCRIPTION
**Pull request by amycsong98**
* fix SimpleCov issues in cucumber
* fix SimpleCov issues in rspec tests
* fix create item form (fix issues with capybara filling in the form)
* create request spec
* update request spec
* destroy request spec
* show request spec

Screen Shot 2020-12-02 at 11 55 27 AM

### CODE REVIEW

File: app/views/layouts/_item_form.html.erb [hunk 1]
```diff
@@ -5,42 +5,30 @@
          <%= form_with model: @item, local: true do |f| %>
             <div class="panel panel-default panel-body">
                <div class="form-group">
-                  <label class="col-sm-2 control-label">Name</label>
```
Comment (amycsong98) [inline app/views/layouts/_item_form.html.erb:8]: this didn't use rails form_helper standards so i needed to change them to follow naming conventions. i also tested in dev and was able to see the form and it worked correctly

File: features/step_definitions/CRUD_user_steps.rb [hunk 2]
```diff
@@ -1,5 +1,4 @@
 require 'simplecov'
-SimpleCov.start
```
Comment (amycsong98) [inline features/step_definitions/CRUD_user_steps.rb:2]: this doesn't go here; it goes it env file so removed it from all cucumber tests

File: app/controllers/items_controller.rb [hunk 3]
```diff
@@ -28,7 +28,7 @@ def edit
   
   def update
     @item = Item.find(params[:id])
-    if (current_user.items.include?(@item) and @item.update_attributes(item_params))
+    if (current_user.items.include?(@item) && @item.update_attributes(item_params))
```
Comment (amycsong98) [inline app/controllers/items_controller.rb:31]: `and` is used for `and return` but in conditions should use `&&`

File: spec/features/users_request_spec.rb [hunk 4]
```diff
@@ -1,6 +1,5 @@
 require 'simplecov'
 require 'login_helper'
-SimpleCov.start
```
Comment (amycsong98) [inline spec/features/users_request_spec.rb:3]: this doesn't belong in rpsec files; it should go once in rails_helper

**Comment by mariamgrig** [comment]
Looks good to me, Amy!

File: app/views/layouts/_item_form.html.erb (hunk 1 above)
Comment (meg-imperato) [reply app/views/layouts/_item_form.html.erb:8]: Good catch!

**Review by meg-imperato — APPROVED** [review]
Looks great!

**Pull request merged**